src/components/button/Button.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import './Button.css'
+
+function Button({handleClick, type, children, disabled}) {
+    return (
+        <button
+            onClick={handleClick}
+            disabled={disabled}
+            type={type}>
```

Comment:
Hier definieer je goed dat het button element een type attribuut kan ontvangen. Echter als je die niet meegeeft wanneer je het component gebruikt (zoals in App.jsx), dan wordt die ook niet toegepast.